src/api/content-rest-api/api/tags.api.ts

```diff
@@ -443,4 +443,18 @@ parameter are returned in addition to those specified in the **fields** paramete
             {}, {}, {}, {}, tags,
             ['application/json'], ['application/json']);
     }
+
+    /**
+     * Assign tags to node. If tag is new then tag is also created additionally, if tag already exists then it is just assigned.
+     *
+     * @param nodeId Id of node to which tags should be assigned.
+     * @param tags List of tags to create and assign or just assign if they already exist.
+     *
+     * @return Promise<TagPaging>
+     */
+    assignTagsToNode(nodeId: string, tags: TagBody[]): Promise<TagPaging> {
```

Comment:
In case if we provide 1 tag, API returns `TagEntry`, nothing breaks in this case?

Reply:
Indeed you are right there is returned different type for single tag... I did not notice that. It does not break anything for my code my type is incorrect so let me correct it

Reply:
done, thank you for finding that